tests/test_evaluators_1d.py

```diff
@@ -169,5 +170,19 @@ def test_bin_equiprobable(self):
         self.assertAlmostEqual(gof_bin_equiprobable, gof_from_binned, 10)
 
 
+class TestUnbinnedMCSampler(unittest.TestCase):
+    def test_unbinnedMCSampler(self):
+        """should return data-- we're testing the GOF of observing one event<data given uniform"""
+        data = np.array([sps.uniform.rvs()])
+        data_generator = sps.uniform.rvs
+
+        def distance_measure(x):
+            return np.mean(x <= data[0])
+
+        e = EvaluatorBaseMCUnbinned(data, data_generator, distance_measure)
+
+        self.assertAlmostEqual(e.get_pvalue(n_toys=100000), data[0], places=2)
```

Comment:
Maybe you could reduce the number of toys and loosen the required precision. In addition you could add an assertion that the p-value is equal to `np.mean(e.fake_gofs)`.

Reply:
places=2 is pretty loose already I felt